holoviews/core/spaces.py

```diff
@@ -413,7 +420,10 @@ class Callable(param.Parameterized):
     when composite objects such as Layouts are returned from the
     callback. This is required for building interactive, linked
     visualizations (for the backends that support them) when returning
-    Layouts, NdLayouts or GridSpace objects.
+    Layouts, NdLayouts or GridSpace objects. When chaining multiple
+    DynamicMaps into a pipeline the link_inputs parameter declares
+    whether a plot created from the Callable owner is linked to objects
+    referenced in the inputs.
```

Comment:
Maybe 'visualization' instead of plot.

How about something like...

'When chaining multiple DynamicMaps into a pipeline, the link_inputs parameter declares whether the visualization generated from this Callable will inherit the linked streams. This parameter is used as a hint by the applicable backend'